sympy/parsing/tests/test_latex.py

```diff
@@ -187,7 +193,10 @@ def test_import():
 def test_parseable():
     from sympy.parsing.latex import parse_latex
     for latex_str, sympy_expr in GOOD_PAIRS:
-        assert parse_latex(latex_str) == sympy_expr
+        try:
+            assert parse_latex(latex_str) == sympy_expr
```

Comment:
You can use

```suggestion
            assert parse_latex(latex_str) == sympy_expr, latex_str + str(sympy_expr)
```

to do this (or if you run the tests using pytest it will show you the variables automatically)